Action/ActionInterface.php

```diff
@@ -65,4 +65,11 @@ public function setFields($fields);
      * @param Field $field
      */
     public function addField(Field $field);
+
+    /**
+     * Return true if the Action requires pagination.
+     *
+     * @return bool
+     */
+    public function requirePagination();
```

Comment:
might be subjective but this function name's suggests that an action is performed. Mb `paginationRequired` instead?